examples/README.md

```diff
@@ -107,3 +119,61 @@ through diagnostic messages it emits to the console, and through the packets
 sniffed up by tcpdump on recipient interfaces.
 I use `sudo tcpdump -vvvnXX -i IF` to see a detailed description of each packet,
 replacing `IF` with the interface you want to sniff on.
+
+# Testing the NAT with Mininet
+[Mininet](http://mininet.org/) is a tool for emulating networks on your local machine.
+This is quite handy because we can run reproducible tests and experiment with
+networks when we don't have the physical machines otherwise needed.
+
+To test the NAT implementation:
+- [Install](http://mininet.org/download/) Mininet - it doesn't matter which way. We used Mininet 2.2.1 on Ubuntu 14.04 LTS.
+- cd into your cloned Pax directory and run `$ sudo ./examples/nat_topo.py test`
+- You can also jump into the Mininet CLI by running `$ sudo ./examples/nat_topo.py`
+
+You will find it very helpful to learn more about Mininet if you plan to do anything
+apart from run the automated test. The Mininet
+[walkthrough](http://mininet.org/walkthrough/) is a pretty good start.
+One very useful thing to know is to use `mininet> host command` in the CLI to run
+`command` on `host`. E.g. to run ifconfig on in1, use `mininet> in1 ifconfig`.
+
+We could manually test the NAT by running these commands:
+``` bash
+$ sudo ./examples/nat_topo.py
+mininet> out0 echo 'Hi from out0!' | netcat -l 12010 &
+mininet> in1 netcat out0 12010
+```
+Note that on line 3, we specify `out0` instead of the IP of out0. We can do this
+because the Mininet CLI replaces it with the IP. Be aware that this is the IP of the
+default interface, which is fine here because out0 only has one, but on nat0
+it might not be the IP you were expecting.
+
+_
+
+Feel free to look in `examples/nat_topo.py`:
+- The `NatTopo` class defines the network topology for Mininet 
+```
+                  ┌──────┐     ┌─────┐
+                  |      |-----│ in1 |
+    ┌──────┐      |      |     └─────┘
+    │ out0 |------| nat0 |
+    └──────┘      |      |     ┌─────┐
+                  |      |-----│ in2 |
+                  └──────┘     └─────┘
+```
+- You might notice that `nat0` is specified as being a node of type `PaxNode`. You can
+  look at the definition in `pax_mininet_node.py`. It is a host with firewall rules on
```

Comment:
`pax_mininet_node.py` could be turned into a link to the file, for added convenience for readers.